### PR TITLE
Additional error checking for focusPointsUsed

### PR DESCRIPTION
**Pull request by project802**
I found files that had no AF point data, so this puts up an error instead of terminating with a nil reference.

### CODE REVIEW

File: focuspoints.lrdevplugin/DefaultPointRenderer.lua [hunk 1]
```diff
@@ -128,6 +135,10 @@ function DefaultPointRenderer.getAutoFocusPoint(metaData)
     focusPointUsed = ExifUtils.findValue( metaData, DefaultPointRenderer.metaPrimaryAFPoint )
   end
 
+  if "(none)" == focusPointUsed then
```
Comment (musselwhizzle) [inline focuspoints.lrdevplugin/DefaultPointRenderer.lua]: i'll play around this with some after work. Question for you, any idea how universal "(none)" will be? Do you think that string will hold up for pentax or canons?

File: focuspoints.lrdevplugin/DefaultPointRenderer.lua [hunk 2]
```diff
@@ -128,6 +135,10 @@ function DefaultPointRenderer.getAutoFocusPoint(metaData)
...
+    focusPointUsed = -1
```
Comment (musselwhizzle) [inline focuspoints.lrdevplugin/DefaultPointRenderer.lua]: nit: let's return nil here. focusPointUsed should be a reference and less of a number.

File: focuspoints.lrdevplugin/DefaultPointRenderer.lua (hunk 1 above)
Comment (project802) [inline focuspoints.lrdevplugin/DefaultPointRenderer.lua]: exiftool uses "(none)" so it should be good as long as the tool stays the same.  There are other ways of launching exiftool that will produce no output at all if the tag isn't populated.  There are other issues with Pentax and Canon as they use the "AFPointsInFocus" tag, which isn't in this plugin yet.  A good entropy of test files will be needed to build out the EXIF parsing logic.

File: focuspoints.lrdevplugin/DefaultPointRenderer.lua (hunk 2 above)
Comment (project802) [inline focuspoints.lrdevplugin/DefaultPointRenderer.lua]: Good idea.

**Comment by project802** [comment]
I was going to take a peek at the rotate math since I did one test and noticed it was off.  I have an idea of how I think it should work in my head, but haven't read the source yet to see how you tackled it.

File: focuspoints.lrdevplugin/DefaultPointRenderer.lua (hunk 1 above)
Comment (musselwhizzle) [inline focuspoints.lrdevplugin/DefaultPointRenderer.lua]: Interesting. Good info. Thanks.

**Comment by musselwhizzle** [comment]
Ok, that's all you if you're up for it. Heads up for you, In the coming days or so I'm going to a little refactor work. It seems Fuji camera give the x,y in the metadata so they dont need a map.

**Comment by project802** [comment]
Yea I expected DefaultPointRenderer.getAutoFocusPoint to change quite a bit as new brands came along and folks uncover new understanding of the camera behavior.  Thats pretty cool of Fuji.

**Comment by musselwhizzle** [comment]
Heads up (since I dont know a better way to let ya know), I've got a PR out here that I will merge in soon. It's a moderate sized refactor. 
https://github.com/musselwhizzle/Focus-Points/pull/7

**Pull request merged**